dist/spec.yaml

```
@@ -11747,6 +11794,11 @@ definitions:
         type: string
     x-okta-tags:
       - UserFactor
+  RawString:
+    type: string
```

```suggestion
    properties:
      responce:
        readOnly: true
        type: string
```
Can we have something like this?